plugins/remark-lint-validate-internal-links/test/index.js

```diff
@@ -0,0 +1,48 @@
+import { resolve } from 'path';
+import { remark } from 'remark';
+import { expect } from 'chai';
+import { readSync } from 'to-vfile';
+import lintRule from '../index.js';
+
+describe('remark-lint-heading-sentence-case', () => {
+	before(() => {
+		process.chdir(resolve(process.cwd(), 'test', 'fixtures'));
+	});
+
+	after(() => {
+		process.chdir(resolve(process.cwd(), '..', '..'));
+	});
+
+	it('should validate links.md', async () => {
+		const expectedWarnings = [
+			"links.md:31:39-31:74: Relative paths must be replaced with abcolute paths from the root e.g. /docs/guides/best_practices",
+			"links.md:32:40-32:86: Relative paths must be replaced with abcolute paths from the root e.g. /docs/guides/best_practices",
+			"links.md:48:39-48:65: Referred doc does not exists or is unnessessary specified with the extension.",
```

Comment:
```suggestion
			"links.md:48:39-48:65: Referred doc does not exists or is unnecessary specified with the extension.",
```
Please change this from L20 - L27